ui/apps/platform/cypress/integration/clusters/delegatedScanning.test.js

```diff
@@ -63,4 +68,24 @@ describe('Delegate Image Scanning', () => {
             '.pf-c-alert.pf-m-success .pf-c-alert__title:contains("Delegated scanning configuration saved successfully")'
         );
     });
+
+    describe('when user does not have permission to see page', () => {
+        it(`should not show the page`, () => {
+            cy.fixture('auth/mypermissionsNoAdminAccess.json').then(({ resourceToAccess }) => {
```

Comment:
For my understanding, would `'auth/mypermissionsMinimalAccess.json` work?

It has `"Administration": "NO_ACCESS"`

I think it corresponds to **Analyst** permission set, or probably should be made to, if it does not.

Reply:
I considered that, but decided that it would be better to create an "almost superuser" who had every permission EXCEPT the one required to see this route.

If we do more of this, we could flip your paradigm of taking a minimal user and adding the required permission for each suite; create a "maximal" permission user (basically and admin) and selectively remove the permission to check the negative case.